Mathlib/Topology/Algebra/Module/Dual.lean

```diff
@@ -0,0 +1,75 @@
+/-
+Copyright (c) 2024 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll, Kalle Kytölä
+-/
+import Mathlib.LinearAlgebra.SesquilinearForm
+import Mathlib.Topology.Algebra.Module.Basic
+
+/-!
+# Topological dual
+
+In a topological vector space `E` the topological dual `TopologicalSpace.Dual 𝕜 E` is the space of
+all continuous linear functions into `𝕜`, `E →L[𝕜] 𝕜`. We define the as an abbreviation, so that it
+automatically inherits the strong topology (the topology of bounded convergence). In particular, if
+`E` is a normed space, then `TopologicalSpace.Dual 𝕜 E` is a normed space as well.
+
+## Main definitions
+
+* `TopologicalSpace.Dual`: abbreviation for `E →L[𝕜] 𝕜`.
```

Comment:
I really don't love this renaming. It's not the dual of a topological space, it's the dual of a topological *vector* space. How about `ContinuousLinearMap.Dual`? It's not perfect either, but I think it captures things better. And probably you'll want that namespace open when working with this anyway.

Same comments apply to the other definitions moved into this namespace.

Reply:
And what about `Topology.Dual`? If you don't like it I'm fine with `ContinuousLinearMap` as well, and we can always change it later.